cookbook/configuration/pdo_session_storage.rst

```diff
@@ -236,6 +216,12 @@ following (MySQL):
         `sess_lifetime` MEDIUMINT NOT NULL
     ) COLLATE utf8_bin, ENGINE = InnoDB;
 
+.. note::
+
+    A ``BLOB`` column type can only store up to 64 kb. If the data stored in
+    a user's session exceeds this, their session will be reset. Consider using
```

Comment:
it might also throw an exception when saving it depending of the way the MySQL server is configured